src/content/reference/fleet-management/clusters/introduction_cluster_api.md

```diff
@@ -0,0 +1,45 @@
+
+Cluster API leverages the concept of operators and Custom Resource Definitions (CRDs) to manage Kubernetes clusters. An operator is essentially a controller that responds to cluster lifecycle events, allowing for automated creation, configuration, and management of clusters.
+
+Custom Resource Definitions (CRDs) are extensions of the Kubernetes API that allow the creation of new resources – in this case, the cluster resources. When a user defines such cluster resources, the Cluster API operator reacts to this declarative statement and works to achieve the desired state of the cluster. This includes tasks such as provisioning infrastructure, bootstrapping nodes, and applying configuration.
+
+## Giant Swarm flavor
+
+The Giant Swarm platform offers additional capabilities on top of Cluster API that we consider as must-have for a complete production system and those are not part of the upstream project, yet.
+
+We leverage these features of Cluster API:
+
+- Certificate management: Cluster API manages important certificates in a Kubernetes cluster, like for kubelet or etcd.
+- Node bootstrap: the operator correctly bootstraps nodes in the cluster from the defined configuration.
+- Infrastructure provisioning: the operator talks to the provider-specfic API (e.g. AWS API) to create the infrastructure needed for the cluster.
+- Cluster autoscaling: Cluster API integrates with the [cluster-autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler) project to allow clusters to grow or shrink based on different policies.
+- Health checks: the operator is able to check the health of the nodes and take actions to avoid manual intervention.
+
+On top of that, we have some additional features that are not part of the Cluster API standard:
+
+- DNS management: we provide a DNS operator to manage DNS assets for the clusters dynamically.
+- Provider credentials management: we have automated the credentials management to operate the provider infrastructure in a secure way.
+- Encryption configuration: we have designed an operator that controls the cryptographic assets for workload clusters and configure them properly making sure they are rotated and updated as needed.
+- Cluster API monitoring: we monitor all the cluster API specific resources letting us know the status of the cluster creation and the health of the cluster.
+- Further, we have some other operators that are provider specific and helps to automate all parts of the cluster lifecycle, like VPC provisioning or security groups management, when upstream Cluster API does not provide it.
```

Comment:
@AndiDog I moved the page under `fleet-management` and the comments went away. Anyway, take a look I have tried to explain every point highlighting the benefit for customers and I added the quote "and those are not part of the upstream project, yet." in the chapter intro